wireless/sims/instance-post-example-1/instance-post-example-1.5.x.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Get the PHP helper library from twilio.com/docs/php/install
+require_once '/path/to/vendor/autoload.php'; // Loads the library
+use Twilio\Rest\Client;
+
+// Your Account Sid and Auth Token from twilio.com/user/account
+$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+$token = "your_auth_token";
+$client = new Client($sid, $token);
+
+$sim = $client->preview->wireless
+    ->sims("DEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA")
+    ->update([
+        'Status' => 'active',
```

Comment:
I believe these options should be lowercase as well so `CallbackUrl` should be `callbackUrl`.